.github/workflows/start-release.yml

```diff
@@ -26,6 +26,11 @@ on:
         type: boolean
         description: Check the box for a dry-run - A dry-run will not push any external state (branches, tags, images, or PyPI packages).
         default: false
+      multi-arch:
```

Comment:
This is currently unused, but will be used if we start triggering the `release.yml` directly from this job. Do we want to leave that around until that time?

Reply:
Gotcha, I figure it's fine to leave around. Ultimately I'm going to remove the option to not build multi-arch (just want a little more bake time) so all the `multi-arch`es are temporary